packages/test-framework/src/fixture/pageObject/Assertions.ts

```diff
@@ -16,6 +16,16 @@ export class Assertions {
         await expect(page.locator('.checkout-step--payment')).toContainText('Payment');
     }
 
+    async shouldSeeElement(element: string): Promise<void> {
+        const page = this.page;
+
+        await expect(page.locator(element)).toBeVisible();
+    }
+
+    shouldMadeRequest(request: boolean) {
+        expect(request).toBeTruthy();
+    }
```

Comment:
This helper function seems not related to reading requests.
Maybe for the usecase of this PR, we can assert the value directly.